velox/exec/tests/utils/TpchQueryBuilder.h

```diff
@@ -105,6 +114,11 @@ class TpchQueryBuilder {
   static const std::unordered_map<std::string, std::vector<std::string>>
       kTables_;
   static const std::vector<std::string> kTableNames_;
+
+  const std::string kLineitem_ = "lineitem";
```

Comment:
To define constants, use this syntax:

```
static const std::string kLineitem = "lineitem";
```

Constant names should not have trailing underscore.